{"text": "2024-09-13 06:35:47 | INFO    | __main__:stash:131 | run - {'image': '20231121', 'version': '0.10.1.post2', 'role': 'EpicPlayer', 'headless': True}\n", "record": {"elapsed": {"repr": "0:00:03.351652", "seconds": 3.351652}, "exception": null, "extra": {"image": "20231121", "version": "0.10.1.post2", "role": "EpicPlayer", "headless": true}, "file": {"name": "claim.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/claim.py"}, "function": "stash", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 131, "message": "run", "module": "claim", "name": "__main__", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:35:47.435180+08:00", "timestamp": 1726180547.43518}}}
{"text": "2024-09-13 06:35:55 | INFO    | __main__:claim_epic_games:97 | Try to flush cookie - {'task': 'claim_epic_games'}\n", "record": {"elapsed": {"repr": "0:00:11.592895", "seconds": 11.592895}, "exception": null, "extra": {"task": "claim_epic_games"}, "file": {"name": "claim.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/claim.py"}, "function": "claim_epic_games", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 97, "message": "Try to flush cookie", "module": "claim", "name": "__main__", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:35:55.676423+08:00", "timestamp": 1726180555.676423}}}
{"text": "2024-09-13 06:36:25 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 0}\n", "record": {"elapsed": {"repr": "0:00:41.830136", "seconds": 41.830136}, "exception": null, "extra": {"task": "authorize", "retry": 0}, "file": {"name": "agent.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:36:25.913664+08:00", "timestamp": 1726180585.913664}}}
{"text": "2024-09-13 06:36:55 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 1}\n", "record": {"elapsed": {"repr": "0:01:11.832453", "seconds": 71.832453}, "exception": null, "extra": {"task": "authorize", "retry": 1}, "file": {"name": "agent.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:36:55.915981+08:00", "timestamp": 1726180615.915981}}}
{"text": "2024-09-13 06:36:58 | INFO    | epic_games.agent:_login:237 | login-with-email - {'url': 'https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games&__cf_chl_rt_tk=b9WGCcYk9bBiNye6spz8QCbtbMxH8NwV4plwaItZv0k-1726180618-0.0.1.1-7572'}\n", "record": {"elapsed": {"repr": "0:01:14.081471", "seconds": 74.081471}, "exception": null, "extra": {"url": "https://www.epicgames.com/id/login?lang=en-US&noHostRedirect=true&redirectUrl=https://store.epicgames.com/en-US/free-games&__cf_chl_rt_tk=b9WGCcYk9bBiNye6spz8QCbtbMxH8NwV4plwaItZv0k-1726180618-0.0.1.1-7572"}, "file": {"name": "agent.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/epic_games/agent.py"}, "function": "_login", "level": {"icon": "ℹ️", "name": "INFO", "no": 20}, "line": 237, "message": "login-with-email", "module": "agent", "name": "epic_games.agent", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:36:58.164999+08:00", "timestamp": 1726180618.164999}}}
{"text": "2024-09-13 06:37:28 | WARNING    | epic_games.agent:authorize:266 | 执行超时 - {'task': 'authorize', 'retry': 2}\n", "record": {"elapsed": {"repr": "0:01:44.082838", "seconds": 104.082838}, "exception": null, "extra": {"task": "authorize", "retry": 2}, "file": {"name": "agent.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/epic_games/agent.py"}, "function": "authorize", "level": {"icon": "⚠️", "name": "WARNING", "no": 30}, "line": 266, "message": "执行超时", "module": "agent", "name": "epic_games.agent", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:37:28.166366+08:00", "timestamp": 1726180648.166366}}}
{"text": "2024-09-13 06:37:28 | ERROR    | __main__:run:158 | An error has been caught in function 'run', process 'MainProcess' (3618), thread 'MainThread' (140152250882944): - {}\nTraceback (most recent call last):\n  File \"/home/runner/work/myEpic/myEpic/epic/src/claim.py\", line 162, in <module>\n    asyncio.run(run())\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/runners.py\", line 44, in run\n    return loop.run_until_complete(main)\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 636, in run_until_complete\n    self.run_forever()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 603, in run_forever\n    self._run_once()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/base_events.py\", line 1909, in _run_once\n    handle._run()\n  File \"/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/asyncio/events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n> File \"/home/runner/work/myEpic/myEpic/epic/src/claim.py\", line 158, in run\n    await agent.stash()\n  File \"/home/runner/work/myEpic/myEpic/epic/src/claim.py\", line 151, in stash\n    await self.claim_epic_games(context)\n  File \"/home/runner/work/myEpic/myEpic/epic/src/claim.py\", line 98, in claim_epic_games\n    if await epic.authorize(page):\n  File \"/home/runner/work/myEpic/myEpic/epic/src/epic_games/agent.py\", line 269, in authorize\n    raise RuntimeError(f\"Failed to flush token - agent={self.__class__.__name__}\")\nRuntimeError: Failed to flush token - agent=EpicGames\n", "record": {"elapsed": {"repr": "0:01:44.311227", "seconds": 104.311227}, "exception": {"type": "RuntimeError", "value": "Failed to flush token - agent=EpicGames", "traceback": true}, "extra": {}, "file": {"name": "claim.py", "path": "/home/runner/work/myEpic/myEpic/epic/src/claim.py"}, "function": "run", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 158, "message": "An error has been caught in function 'run', process 'MainProcess' (3618), thread 'MainThread' (140152250882944):", "module": "claim", "name": "__main__", "process": {"id": 3618, "name": "MainProcess"}, "thread": {"id": 140152250882944, "name": "MainThread"}, "time": {"repr": "2024-09-13 06:37:28.394755+08:00", "timestamp": 1726180648.394755}}}
